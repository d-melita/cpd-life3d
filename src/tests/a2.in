1000 64 0.4 0
