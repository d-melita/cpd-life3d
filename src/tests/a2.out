1 81443 62
2 24563 24
3 20080 1
4 19016 1
5 17576 1
6 16905 1
7 15793 1
8 15174 1
9 14807 1
